packages/solana/exports/vue.ts

```diff
@@ -0,0 +1,124 @@
+import { useSnapshot } from 'valtio'
```

Comment:
Valtio is a React dependent package and should not be used here. Check the `scaffold-vue` package please.